sale_exception_max_allowed_discount/README.rst

```diff
@@ -0,0 +1,52 @@
+Sale exception maximum allowed discount
+=======================================
+
+This module was created to extend the sales process. It allows to
+set a maximum permitted sales discount % on each product on a sale quotation.
+
+A sales user cannot approve a sales quotation if any of the items contain a
+sales discount % above the maximum allowed.
+
+Installation
+============
+
+No specific installation steps are required.
+
+Configuration
+=============
+
+No specific configuration steps are required.
```

Comment:
Where do you specify the max allowed discount? You should note it in this section.